dotnet-interactive.Tests/CommandLine/CommandLineParserTests.cs

```diff
@@ -91,6 +93,18 @@ public void jupyter_command_parses_port_range_option()
                 .BeEquivalentTo(new PortRange(3000, 4000));
         }
 
+        [Fact]
+        public void jupyter_install_command_parses_path_option()
+        {
+            Directory.CreateDirectory(_kernelSpecInstallPath.FullName);
+            
+            var result = _parser.Parse($"jupyter install --path {_kernelSpecInstallPath}");
+
+            var option = result.CommandResult.OptionResult("--path");
+
+            option.Should().NotBeNull();
```

Comment:
We should also verify that the path was captured correctly. Something like this:

```csharp

result.FindResultFor(option).GetValueOrDefault().Should().Be(...)

```